examples/basic.ts

```diff
@@ -1,8 +1,8 @@
 import {Sequelize, DataTypes} from 'sequelize';
 import { Configurations, CredentialProvider } from "@gomomento/sdk";
-import { MomentoClientGenerator } from "../src/cacheclient/momento/momento-client-generator";
-import { LoggerFactory } from "../src/logger/logger-factory";
-import {modelCacheFactory} from "../src/model-cache-factory";
+import { MomentoClientGenerator } from "../src";
+import { LoggerFactory } from "../src";
+import { modelCacheFactory } from "../src";
```

Comment:
these should say `from '@gomomento-poc/momento-sequelize-cache'`, shouldn't they?

Reply:
yes once we release again as now there's no `index.ts`.. i got module not found if i did `@gomomento-poc/momento-sequelize-cache/full-path`